src/helpers/queryDataParser.js

```diff
@@ -0,0 +1,151 @@
+import _filter from 'lodash/filter';
+import { texts } from '../config';
+
+import { QUERY_TYPES } from '../queries';
+import { eventDate } from './dateTimeHelper';
+import { mainImageOfMediaContents } from './imageHelper';
+import { momentFormat } from './momentHelper';
+import { shareMessage } from './shareHelper';
+import { subtitle } from './textHelper';
+
+export const parseEventRecords = (data, skipLastDivider = false) => {
+  return data?.map((eventRecord, index) => ({
+    id: eventRecord.id,
+    subtitle: subtitle(
+      eventDate(eventRecord.listDate),
+      eventRecord.addresses?.[0]?.addition || eventRecord.addresses?.[0]?.city
+    ),
+    title: eventRecord.title,
+    picture: {
+      url: mainImageOfMediaContents(eventRecord.mediaContents)
+    },
+    routeName: 'Detail',
+    params: {
+      title: 'Veranstaltung',
+      query: QUERY_TYPES.EVENT_RECORD,
+      queryVariables: { id: `${eventRecord.id}` },
+      rootRouteName: 'EventRecords',
+      shareContent: {
+        message: shareMessage(eventRecord, QUERY_TYPES.EVENT_RECORD)
+      },
+      details: eventRecord
+    },
+    bottomDivider: skipLastDivider ? index !== data.length - 1 : undefined
```

Comment:
we can use following for `bottomDivider` in all parse methods to implicitly return `undefined` which simplifies in my eyes:

```
bottomDivider: skipLastDivider && index !== data.length - 1
```

what do you think?

Reply:
I changed it in 1ba7bec2cc34621ee768b59c4fb706fa19475e62

But on a side note: why do we use bottom dividers on the list entries and not top dividers?

By using top dividers we could enable them on each item without the need to check for indices if I see it correctly.

that would also fix the inconsistency with tours and POIs.

Reply:
if the first item has a top divider, it looks weird combined with the <TitleContainer><Title>... so we would need to have to condition other way around.